app/controllers/concerns/course/statistics/live_feedback_concern.rb

```diff
@@ -2,11 +2,42 @@
 module Course::Statistics::LiveFeedbackConcern
   private
 
+  def find_submitter_course_user(submission)
```

Comment:
Put all the changes in this file on this commit https://github.com/Coursemology/coursemology2/pull/7497/commits/d7f986c54545a036cff8f6a821ac6b37754e16cb

Reply:
done